playwright/auth/auth-index.test.ts

```diff
@@ -1,5 +1,131 @@
+import { expect, test } from "@playwright/test";
+
+import { BASE_URL } from "@lib/config/constants";
+import prisma from "@lib/prisma";
+
 import { todo } from "../lib/testUtils";
 
-todo("Can signup from a team invite");
+test.describe("Can signup from a team invite", async () => {
+  let page;
+  let token: string | undefined;
+  let signupFromInviteURL = "";
+  const team = "test-team";
+  const email = "test@test.com";
+  const password = "secretpassword123";
+  const validUsername = "test-user";
+  const usernameAlreadyTaken = "pro";
+  const emailAlreadyTaken = "pro@example.com";
+
+  test.use({ storageState: "playwright/artifacts/proStorageState.json" });
```

Comment:
Ok. Upon further inspection, I think it makes more sense to use the seeded `teampro` account instead of the `pro` account. This so we don't require to reset the DB each time since the `teampro` account already has a team enabled.

We can achieve this by uncommenting this line:

https://github.com/calendso/calendso/blob/ce4ed40ae0818b67bb4a937ba8ec738e888a6003/playwright/lib/globalSetup.ts#L36

An then using those credentials instead:

```suggestion
  test.use({ storageState: "playwright/artifacts/teamproStorageState.json" });
```

Reply:
Sounds good! I will take a look

Reply:
Done 🙆🏻‍♂️ @zomars